prisonstaffhub-specs/src/test/groovy/uk/gov/justice/digital/hmpps/prisonstaffhub/pages/AdjudicationHistoryPage.groovy

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.digital.hmpps.prisonstaffhub.pages
+
+import geb.Page
+
+class AdjudicationHistoryPage extends Page {
+    static url = "/global-search-results"
+
```

Comment:
Is this the right url for the page?

Reply:
no it's not - I've removed this as it's not being used